docs/glossary.md

```diff
@@ -1,62 +1,84 @@
-**Acronyms**: CSV(s)
+**Aliases**: CSV(s)
+
+### CatalogSources
+
+**Definition**: The CatalogSource represents a repository of bundles, which are collections of resources that must contain [CSVs](#ClusterServiceVersion), [CRDs](#CustomResourceDefinitions), and package definitions. There are multiple implementations of a CatalogSource backend, the current recommendation is to use a [registry image](#Index).
+**Aliases**: CatSrc(s)
 
 ### InstallPlans
 
-**Definition**: The InstallPlan lists out all resources to be created in order to automatically install or upgrade a [CSV](#ClusterServiceVersion). 
-**Acronyms**: IP(s)
+**Definition**: The InstallPlan defines a set of resources to be created in order to install or upgrade to a specific version of [CSV](#ClusterServiceVersion). 
+**Aliases**: IP(s)
 
 ### OperatorGroups
 
-**Definition**: The operator group selects a set of target namespaces in which to generate required RBAC access for its member Operators.
-**Acronyms**: OG(s)
+**Definition**: The OperatorGroup selects a set of target namespaces in which to generate required RBAC access for its member Operators.
+**Aliases**: OG(s)
 
 ### OperatorSources
 
-**Definition**: The operator source defines the external data store used to store Operator bundles. It contains information about the catalog of operators including its name, type, namespace, and display name. 
-**Acronyms**: OpSrc(s)
+**Definition**: The OperatorSources are external app registries that contains a catalog of operators. Applying an OperatorSource to a cluster makes the operators in that OperatorSource available for installation in that cluster.
+**Aliases**: OpSrc(s)
 
 ### Subscriptions
 
-**Definition**: The subscription defines the channel and the source of Operator updates and is used to keep the CSV updated by tracking a channel in a package. 
-**Acronyms**: Subs(s)
+**Definition**: The Subscription defines the channel and the source of Operator updates and is used to keep the CSV updated by binding with a channel in a package. 
+**Aliases**: Subs(s)
 
 ## OLM Concepts
 
 ### Bundle
 
-**Definition**: A collection of Operator manifests and metadata which together form a unique version of an Operator that can be installed onto the cluster. 
-**Acronyms**: Package
+**Definition**: A collection of Operator [CSV](#ClusterServiceVersion), manifests, and metadata which together form a unique version of an Operator that can be installed onto the cluster. 
+**Aliases**:
+
+### Bundle Image
+
+**Definition**: An image of a bundle is built from operator manifests and contains exactly one [bundle](#Bundle). The bundle images are stored and distributed by app registries such as Quay.io or DockerHub.
+**Aliases**:
```

Comment:
Super nit: Is this necessary given that no aliases exist? Applies in all empty Alias lists.

Reply:
+1, it seems like this field should only be here when the glossary entry is a CRD.